vcd/config_test.go

```diff
@@ -93,6 +93,24 @@ type TestConfig struct {
 		UploadPieceSize int64  `json:"uploadPieceSize,omitempty"`
 		UploadProgress  bool   `json:"uploadProgress,omitempty"`
 	} `json:"media"`
+	// Data used to create a new environment, in addition to the regular test configuration file
+	TestEnvBuild struct {
+		Gateway                      string `json:"gateway"`                      // Gateway for external network
+		Netmask                      string `json:"netmask"`                      // Netmask for external network
+		ExternalNetworkStaticStartIp string `json:"externalNetworkStaticStartIp"` // Start IP for external network
+		ExternalNetworkStaticEndIp   string `json:"externalNetworkStaticEndIp"`   // End IP for external network
```

Comment:
Thanks for making these consistent. Suggesting of shortening the names:
```
ExternalNetworkStartIp string `json:"externalNetworkStartIp"` // Start IP for external network
ExternalNetworkEndIp   string `json:"externalNetworkEndIp"`   // End IP for external network
```

Reply:
Done